disnake/guild.py

```diff
@@ -202,6 +207,7 @@ class Guild(Hashable):
 
         - ``ANIMATED_BANNER``: Guild can upload an animated banner.
         - ``ANIMATED_ICON``: Guild can upload an animated icon.
+        - ``AUTO_MODERATION``: Guild has enabled auto moderation.
```

Comment:
NIT: Guild has set up auto moderation rules.

The feature exists if the guild has any rule, not whether they are disabled or not.